snyk-monitor/templates/deployment.yaml

```diff
@@ -188,18 +188,24 @@ spec:
           - name: SNYK_WORKERS_COUNT
             value: {{ quote .Values.workers.count }}
           {{- if .Values.sysdig.enabled }}
-          - name: SNYK_SYSDIG_ENDPOINT
+          - name: SNYK_SYSDIG_REGION_URL
             valueFrom:
               secretKeyRef:
                 name: {{ .Values.sysdig.secretName }}
-                key: endpoint
+                key: region
                 optional: true
-          - name: SNYK_SYSDIG_TOKEN
+          - name: SNYK_SYSDIG_API_TOKEN
             valueFrom:
               secretKeyRef:
                 name: {{ .Values.sysdig.secretName }}
                 key: token
                 optional: true
+          - name: SNYK_SYSDIG_CLUSTER_NAME
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.sysdig.secretName }}
+                key: cluster
+                optional: true
```

Comment:
are they optional though?

Reply:
They should be there, unless a further check fails anyway, if they're not configure.
I kept it optional as it was for the previous secret... Any idea why it'd have been optional before?

Reply:
I don't think we should introduce a breaking change by making them non-optional. If some customers currently have `sysdig.enabled` set to `true` but haven't got around to actually adding the secrets for whatever reason, we break the entire monitor by making them non-optional.
I agree that it probably would've helped customers debug things easier if they weren't optional, but I don't think now is the time to change it.

Reply:
why not? this is a breaking change anyway

Reply:
to customers with a fully functioning sysdig integration yes, but not everyone

Reply:
do we really need to account for customers who have set the `sysdig.enabled` to `true` but are not actually using sysdig? seems a bit of a weird edge case

Reply:
If even one exists, we might avoid a support ticket by keeping them optional!
My rationale here is that because this is a short term solution, we should limit the impact of this change as much as possible.

Reply:
we might avoid a support ticket by making them not optional 😆 anyways I don't mind